gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started-kubernetes.adoc

```diff
@@ -0,0 +1,421 @@
++
+----
+Starting registration, please be patient.
+Registration completed, the configuration files were written 
+in directory "/registration". For security, modify the file permissions 
+to restrict production scenario access to the user running flex.
+----
++
+The `Registration completed` message in the terminal window refers to a `/registration` directory that resides within the Docker container.  
++
+Use `sudo` if you encounter file permission issues when running this command. If your token expires before you run the command, a `400 Bad Request` error occurs, so you need to refresh the Runtime Manager page to generate a new token and run the modified command that appears on the page.
+
+
+== Step 3: Create a Repository for the Helm Chart
+
+Add and update a local repository that stores a Helm chart with your Flex Gateway replica's configuration. Note that you can get more information about the properties of the Helm chart for Flex Gateway in https://artifacthub.io/packages/helm/flex-gateway/flex-gateway[ArtifactHUB^].
+
+. Add the Helm repo: 
++
+[source,helm]
+----
+helm repo add flex-gateway https://flex-packages.anypoint.mulesoft.com/helm
+----
++
+The command either creates a repository or skips the repository creation process if one exists already:
++
+* If the repository is new, the command returns the following message:
++
+----
+"flex-gateway" has been added to your repositories
+----
++
+* If the repository exists already, the command returns the following message:
++
+----
+"flex-gateway" already exists with the same configuration, skipping
+----
+
+. Run `helm repo up`.
++
+The command returns the following message:
++
+----
+Hang tight while we grab the latest from your chart repositories...
+...Successfully got an update from the "flex-gateway" chart repository
+Update Complete. ⎈Happy Helming!⎈
+----
+
+[[deploy-flex]]
+== Step 4: Deploy Flex Gateway
+
+Use Helm to deploy a Flex Gateway container (replica) to a node in your cluster and connect it to Anypoint Platform. After deploying, you can use Runtime Manager to verify that the replica of your Flex Gateway is present and connected to Anypoint Platform. 
+
+. From your terminal window, use `kubectl` to point to the cluster into which you plan to deploy the Flex Gateway replica, for example: 
++
+.. List available clusters.
++
+[source,kubectl]
+----
+kubectl config get-contexts
+----
+.. From the list of contexts, identify the cluster to host your Flex Gateway replica.
+.. Select the cluster to use, replacing `my-cluster` in the following example with the cluster you choose: 
++
+[source,kubectl]
+----
+kubectl config use-context my-cluster
+----
++
+For more guidance, see <<use-cluster>>.
+
+. Run the Helm command for deploying your Flex Gateway replica and connecting it to Anypoint Platform. 
++
+[source,helm]
+----
+helm -n gateway upgrade -i --create-namespace --wait ingress flex-gateway/flex-gateway --set-file registration.content=registration.yaml
+----
++
```

Comment:
TODO (NOTE TO SELF): add the results of the command and point to troubleshooting for potential issues

Reply:
done